b2sdk/version.py

```diff
@@ -18,3 +23,178 @@
 PYTHON_VERSION = '.'.join(map(str, sys.version_info[:3]))  # something like: 2.7.11
 
 USER_AGENT = 'backblaze-b2/%s python/%s' % (VERSION, PYTHON_VERSION)
+
+
+class Deprecat:
```

Comment:
I'm lost.  How about a doc string for the class.  What does it do?

Reply:
It looks like you decorate a method with `Deprecat`, and it will issue warnings when you call it.